apps/opik-backend/src/main/java/com/comet/opik/infrastructure/ConfigurationModule.java

```diff
@@ -0,0 +1,13 @@
+package com.comet.opik.infrastructure;
+
+import ru.vyarus.dropwizard.guice.module.support.DropwizardAwareModule;
+
+public class ConfigurationModule extends DropwizardAwareModule<OpikConfiguration> {
```

Comment:
I don't get the need of this module. We're already injecting the YAML configuration everywhere without adding boiler plate code.

Reply:
If you remember, configuration injection only works in modules. For Classes/Components, we need to add the configuration to the context.